tests/unit/insertTestResult.unitTest.ts

```diff
@@ -2279,6 +2279,81 @@ describe("insertTestResult", () => {
         });
     });
 
+    context("when inserting a testResult for a motorcycle", () => {
+        it("should fail if the vehicleClass is missing", () => {
+            const motorcycleWithoutVehicleClass = cloneDeep(testResultsPostMock[12]);
+            delete motorcycleWithoutVehicleClass.vehicleClass;
+            MockTestResultsDAO = jest.fn().mockImplementation(() => {
+                return {
+                    createSingle: () => {
+                        return Promise.resolve(Array.of(motorcycleWithoutVehicleClass));
+                    },
+                    getTestNumber: () => {
+                        return Promise.resolve({
+                            testNumber: "W01A00209",
+                            id: "W01",
+                            certLetter: "A",
+                            sequenceNumber: "002"
+                        });
+                    },
+                    getTestCodesAndClassificationFromTestTypes: () => {
+                        return Promise.resolve({
+                            linkedTestCode: null,
+                            defaultTestCode: "yf4",
+                            testTypeClassification: "Annual NO CERTIFICATE"
```

Comment:
I think these could be moved into a beforeEach block as the mocks are reused for the other test, we would save some space. What do you think?

Reply:
I completely agree with you. I already moved some of these initialisations in one of my previous PRs. Then I started having an issue ( I cannot recall exactly what was the issue ) and I ended up rolling back my change and using this pattern. Probably I can dedicate it some more time now and see why it wasn't working.

Reply:
Ok, changes done as discussed